### PR TITLE
adds node null check before blur event

### LINKED ISSUE #735: [0.17.2] Input blur event cause "Cannot read properties of null (reading 'store')"
### Reproduction

https://stackblitz.com/edit/vitejs-vite-mbv6sq?file=src%2Fpages%2FForm2.vue

### Describe the bug

When leaving a page with a Formkit input focused (using browser navigation to avoid blur before leaving), the blur event occur but the node is already destroyed and it throw this error :

image

This issue is likely due to the absence of verification for the existence of the node in this code.

image

### Environment

• OS: MacOS Ventura (Apple Silicon), Debian 11
• Browser: Chrome
• Version: 113

### PR DESCRIPTION
**Pull request by parafeu**
fix #735

### CODE REVIEW

File: packages/vue/src/bindings.ts [hunk 1]
```diff
@@ -227,7 +227,7 @@ const vueBindings: FormKitPlugin = function vueBindings(node) {
     },
     handlers: {
       blur: (e?: Event) => {
-        if (typeof node === 'undefined') return
+        if (typeof node === 'undefined' || node === null) return
         node.store.set(
           createMessage({ key: 'blurred', visible: false, value: true })
         )
```
Comment (darthf1) [inline packages/vue/src/bindings.ts:233]:
```suggestion
        node?.store.set(
          createMessage({ key: 'blurred', visible: false, value: true })
        )
```

File: packages/vue/src/bindings.ts [hunk 2]
```diff
@@ -227,7 +227,7 @@ const vueBindings: FormKitPlugin = function vueBindings(node) {
     },
     handlers: {
       blur: (e?: Event) => {
-        if (typeof node === 'undefined') return
+        if (typeof node === 'undefined' || node === null) return
```
Comment (GustavoFenilli) [inline packages/vue/src/bindings.ts:230]: If we should check indeed both I think `if (!node) return` should work fine.

**Review by gbyesiltas — APPROVED** [review]
🤓

**Comment by frasza** [comment]
LGTM. Can we get this small but important fix merged and plan a release for people with issues, @GustavoFenilli @justin-schroeder? Thank you!

**Pull request merged**